tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -70,9 +70,9 @@
 
     <!-- MySql.Data .NET/Core references -->
     <PackageReference Include="MySql.Data" Version="6.10.7" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
-    <PackageReference Include ="MySql.Data" Version="8.0.15" Condition="'$(TargetFramework)' == 'net5.0'" />
-    <PackageReference Include ="MySql.Data" Version="8.0.30" Condition="'$(TargetFramework)' == 'net6.0'" />
-    <PackageReference Include ="MySql.Data" Version="8.0.32.1" Condition="'$(TargetFramework)' == 'net7.0'" />
+    <PackageReference Include="MySql.Data" Version="8.0.15" Condition="'$(TargetFramework)' == 'net5.0'" />
```

Comment:
Were these changes intentional?

Reply:
Not intentional, but they're just whitespace changes that VS did automatically.  The pattern in the rest of the file is for there to be no space between `Include` and `=` so I think we should do it.